release-notes/opensearch-skills.release-notes-2.17.0.0.md

```diff
@@ -0,0 +1,13 @@
+# 2024-09-07 Version 2.17.0.0
+
+Compatible with OpenSearch 2.17.0
+
+### Dependencies
```

Comment:
I dont think we have dependencies sections. Should be under maintainence?
https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md

Reply:
Lets change this to `Maintenance`

Reply:
make sense, changing it.